tests/Agent/IntegrationTests/ContainerApplications/MemcachedTestApp/BlogPostService.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System;
+
+namespace MemcachedTestApp
+{
+    public class BlogPostService : IBlogPostService
+    {
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
+        public async Task<Dictionary<string, List<BlogPost>>> GetRecent(int itemCount)
```

Comment:
Why is this an async method?

Reply:
I was having trouble finding the incantation needed to get the client to work without it being async.  I think I might have gotten it working though.